ecosystem/sep-0041.md

```diff
@@ -3,11 +3,11 @@
 ```
 SEP: 0041
 Title: Soroban Token Interface
-Authors: Jonathan Jove <@jonjove>, Siddharth Suresh <@sisuresh>, Simon Chow <@chowbao>
+Authors: Jonathan Jove <@jonjove>, Siddharth Suresh <@sisuresh>, Simon Chow <@chowbao>, Leigh McCulloch <@leighmcculloch>
 Status: Draft
```

Comment:
At what point should we finalized this SEP?

Reply:
+1 A question that needs an answer, but lets discuss that independent of this change.